tokio/src/sync/watch.rs

```diff
@@ -240,9 +233,8 @@ impl<T> Receiver<T> {
         let state = self.shared.version.load(SeqCst);
         let version = state & !CLOSED;
 
-        if version != self.ver {
+        if self.inner.version.swap(version, SeqCst) != version {
```

Comment:
At the very least, if we don't want to introduce unsafe, I think it would be fine to make this `Relaxed` — AFAICT it is never actually modified concurrently?

Reply:
Changed to a `Relaxed` ordering (I doubt unsafe would be worth it).

Reply:
no `unsafe` here :) `Relaxed` is fine